### PR TITLE
operator: Fix corrupted csv and update operator version to 0.23.0

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
Fix:
PR #753 had a huge mistake that changed operator manifest yaml file.
Some part was unintentionally copied and pasted, and no one noticed.
Therefore, this commit replaces the yaml file with the command "operator-sdk generate".

Update:
v0.21.0 -> v0.23.0

Signed-off-by: Hyeongju Johannes Lee <hyeongju.lee@intel.com>

### CODE REVIEW

File: deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -5,9 +5,11 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     categories: Drivers and plugins
-    containerImage: intel/intel-deviceplugin-operator:0.21.0
-    createdAt: "2021-08-18"
-    description: This One Operator is a Kubernetes custom controller whose goal is to serve the installation and lifecycle management of Intel device plugins for Kubernetes.
+    containerImage: intel/intel-deviceplugin-operator:0.23.0
+    createdAt: "2022-01-04"
+    description: This One Operator is a Kubernetes custom controller whose goal is
```
Comment (mythi) [inline deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml:10]: not related to this PR but let reword the description a bit
```suggestion
    description: This operator is a Kubernetes custom controller whose goal is
```

**Review by mythi — COMMENTED** [review]
right now this is "fix and update" because it moves to 0.23.0 in the same commit. my preference is to split them.

File: deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (hj-johannes-lee) [reply deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml:10]: Yeap, I already knew someone would say to divide them into two.! But, I wanted to make sure whether it is the matter of one pr with two commits or two different prs..! You mean two different commits in this pr, right?

And,,, it seems that release-0.23 branch includes the non-working yaml file.. Do we need to do something for that branch also?

Comment (mythi) [reply deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml:10]: it's good now. can you take my suggestion in 25eaf91 as well?

Comment (hj-johannes-lee) [reply deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml]: @mythi I did.! What do you think about deployments/operator/samples/deviceplugin_v1_xxxdeviceplugin.yaml files?

Comment (mythi) [reply deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml]: What about them?

Comment (hj-johannes-lee) [reply deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml]: @mythi If I understood correctly,, I had to change all samples/yaml files, makefile, and webhook_common.go also..!?
If not, please let me know.!

Comment (mythi) [reply deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml]: samples and webhook_common.go are not related to this work so they could go separate from this PR or at least in their own commit

File: Makefile [hunk 2]
```diff
@@ -12,9 +12,9 @@ BUILDER ?= "docker"
 EXTRA_BUILD_ARGS ?= ""
 
 # Current Operator version
-OPERATOR_VERSION ?= 0.21.0
+OPERATOR_VERSION ?= 0.23.0
 # Previous Operator version
-OPERATOR_PREVIOUS_VERSION ?= 0.20.0
+OPERATOR_PREVIOUS_VERSION ?= 0.22.0
```
Comment (mythi) [inline Makefile:17]: prev version is `0.21.0`

**Review by mythi — APPROVED** [review]

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**